ride/manager_vault.ride

```diff
@@ -0,0 +1,243 @@
+{-# STDLIB_VERSION 6 #-}
+{-# CONTENT_TYPE DAPP #-}
+{-# SCRIPT_TYPE ACCOUNT #-}
+
+let SEP = "__"
+let contractFilename = "manager_vault.ride"
+let MAX_ADMIN_LIST_SIZE = 10
+let MIN_ADMIN_LIST_SIZE = 3
+let ADDRESS_BYTES_LENGTH = 26
+let TXID_BYTES_LENGTH = 32
+
+func throwErr(s: String) = {
+  throw(contractFilename + ": " + s)
+}
+
+# Manager Public Key
+func keyActiveManagerPublicKey()  = makeString(["%s", "managerPublicKey"], SEP)
+func keyPendingManagerPublicKey() = makeString(["%s", "pendingManagerPublicKey"], SEP)
+func keyCurrentManagerPublicKey() = makeString(["%s", "currentManagerPublicKey"], SEP)
+
+# Voting prefixes
+func keySetNewManagerVotePrefix(managerPublicKey: String) = makeString(["%s%s%s", "setManager", managerPublicKey], SEP)
+func keyDeactivateManagerVotePrefix()                     = makeString(["%s%s", "deactivateManager"], SEP)
+func keyActivateManagerVotePrefix()                       = makeString(["%s%s", "activateManager"], SEP)
+func keyAddNewAdminVotePrefix(adminCandidate: String)     = makeString(["%s%s%s", "addAdmin", adminCandidate], SEP)
+func keyRemoveAdminVotePrefix(removeCandidate: String)    = makeString(["%s%s%s", "removeAdmin", removeCandidate], SEP)
+func keyAllowedTxIdVotePrefix(txId: String)               = makeString(["%s%s%s", "allowTxId", txId], SEP)
```

Comment:
one `%s` is unnecessary

Reply:
Prefixes is later concatenated with admin address to make full vote key
`"%s%s__activateManager" + "__" + "{adminAddress}"`
Added explicit `keyFullAdminVote(prefix: String, adminAddress: String)` function